helix-term/src/commands/lsp.rs

```diff
@@ -1078,6 +1089,14 @@ pub fn goto_implementation(cx: &mut Context) {
 }
 
 pub fn goto_reference(cx: &mut Context) {
+    goto_reference_impl(cx, false)
+}
+
+pub fn goto_reference_other_window(cx: &mut Context) {
+    goto_reference_impl(cx, true)
+}
+
+fn goto_reference_impl(cx: &mut Context, rotate: bool) {
```

Comment:
nit: `rotate` is unused here (noticed thanks to the Clippy CI step which failed because of this)